src/js/firstparties/facebookcorewwwi.onion.js

```diff
@@ -0,0 +1,79 @@
+// Adapted from https://github.com/mgziminsky/FacebookTrackingRemoval
+(function() {
+let fbtor_wrapped_link = "a[href*='facebookcorewwwi.onion/l.php?'";
```

Comment:
You shouldn't need a separate file for every domain. Instead, use `` `a[href*='${location.origin}/l.php?']` `` in the main facebook file and add the additional domains to the same content script object. This is the approach I plan to take, but I haven't tested this case yet. I do this in a different selector already though, and it was just an oversight on my part that I didn't use `location.origin` on this selector that you copied from.